platform/docs/src/main/resources/IntContents.adoc

```diff
@@ -1,19 +1,17 @@
 
-== Overview
+This section supports integration of this application with external frameworks.
 
-This page supports integration of this application with external frameworks.
-
-== Platform Global Settings
+=== Platform Global Settings
```

Comment:
I think I'll be rewriting this entire section as part of https://codice.atlassian.net/browse/DDF-1775.